tests/integration/cmd_url_test.go

```diff
@@ -131,7 +131,7 @@ var _ = Describe("odo url command tests", func() {
 			// get the route URL
 			fullURLPath := helper.DetermineRouteURL(context)
 			pathNoHTTP := strings.Split(fullURLPath, "//")[1]
-			desiredURLDescribeJSON = fmt.Sprintf(`{"kind":"url","apiVersion":"odo.dev/v1alpha1","metadata":{ "name": "myurl","creationTimestamp": null},"spec":{"host":"%s","protocol": "https","port": 8080,"secure": true,"kind": "route"},"status": {"state": "Pushed"}}`, pathNoHTTP)
+			desiredURLDescribeJSON = fmt.Sprintf(`{"kind":"url","apiVersion":"odo.dev/v1alpha1","metadata":{ "name": "myurl","creationTimestamp": null},"spec":{"host":"%s","protocol": "https","port": 8080,"secure": true, "path": "/", "kind": "route"},"status": {"state": "Pushed"}}`, pathNoHTTP)
```

Comment:
I doubt the ```"path": "/"``` will be same on windows platform.

Reply:
> I doubt the "path": "/" will be same on windows platform.

This is a path part of an URL so it will be the same everywhere.